.appveyor.yml

```diff
@@ -35,4 +35,8 @@ before_build:
   - cd C:\projects\metabench
 
 build_script:
-  - bash tools\run_tests.sh -DRUBY_LIBRARY=%RUBY_LIBRARY%
+  - mkdir build
+  - cd build
+  - cmake .. -G%CMAKE_GENERATOR% -DRUBY_LIBRARY=%RUBY_LIBRARY% -DMETABENCH_UNIT_TEST_OPTIONS="-DRUBY_LIBRARY=%RUBY_LIBRARY%"
```

Comment:
What's up with this `RUBY_LIBRARY` thing, shouldn't we just make sure it's in `PATH`?